src/server/pachyderm_test.go

```diff
@@ -10622,20 +10622,18 @@ func TestSpout(t *testing.T) {
 		iter, err := c.SubscribeCommit(pipeline, "master", nil, "", pfs.CommitState_FINISHED)
 		require.NoError(t, err)
 
-		var files []*pfs.FileInfo
-		// the test image generated from ./etc/testing/spout/main.go sends 1000 files
-		for i := 0; len(files) < 1000; i++ {
+		for i := 0; i < 10; i++ {
```

Comment:
We should fix this test to cycle through all the 1000 files.

Reply:
I'll do that separately for the master branch